docs/devices/324131092621.md

```diff
@@ -86,6 +86,22 @@ Value can be found in the published state on the `action` property.
 It's not possible to read (`/get`) or write (`/set`) this value.
 The possible values are: `on-press`, `on-hold`, `on-hold-release`, `up-press`, `up-hold`, `up-hold-release`, `down-press`, `down-hold`, `down-hold-release`, `off-press`, `off-hold`, `off-hold-release`.
 
+### Brightness (numeric)
```

Comment:
These attributes are deprecated and are available under the `action_*` after setting legacy false for this device.

e.g.

```
devices:
  '0x12345678':
    friendly_name: my_hue_dimmer
    legacy: false
```